data-prepper-core/src/main/java/org/opensearch/dataprepper/peerforwarder/RemotePeerForwarder.java

```diff
@@ -31,16 +29,13 @@
 
 class RemotePeerForwarder implements PeerForwarder {
     private static final Logger LOG = LoggerFactory.getLogger(RemotePeerForwarder.class);
-    private static final int READ_BATCH_DELAY = 3_000;
```

Comment:
Why did you remove this line in this PR? Doesn't #2159 do this?

Reply:
I think removing this was a subtle miss from #2159. It wasn't removed in that PR even though it is no longer used
https://github.com/opensearch-project/data-prepper/pull/2159/files#diff-1cbb2edcb637c79fa75c0ee50aee7d939632e0fa53289bb31da9dbd823bd4e85R34